scout/commands/update/genes.py

```diff
@@ -133,12 +133,12 @@ def genes(build, downloads_folder, api_key):
 
     # If required resources are missing, download them to a temporary file
     if downloads_folder is None:
-        with tempfile.TemporaryDirectory() as tempdir:
+        with tempfile.TemporaryDirectory(dir=os.getcwd()) as tempdir:
```

Comment:
Are there any reasons not to honor the ENV TEMP variables? Say by using tempfile.gettempdir() or such?

Reply:
Document otherwise; temporary files in CWD is not really expected. 😊

Reply:
There's nothing wrong with having the temp dir in ENV TEMP. It's just that in cwd you actually see what is going on during the command execution, and then you see that the temp dir is gone. I can add a comment!

Reply:
I get frustrated with apps that dont listen to any TMPDIR etc. 😆 Perhaps add it to the admin docs and the help text for the command methinks - it is not a common practice.

Reply:
Or I just remove that dir=os.getcwd(), I don't want to make it more complicated than it really is. It's a dir that is going to be removed as soon as the command as finished anyway